src/libraries/System.Diagnostics.TraceSource/src/System.Diagnostics.TraceSource.csproj

```diff
@@ -34,8 +34,9 @@
     <ProjectReference Include="$(LibrariesProjectRoot)System.Collections.Specialized\src\System.Collections.Specialized.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.ComponentModel.Primitives\src\System.ComponentModel.Primitives.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime\src\System.Runtime.csproj" />
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.Extensions\src\System.Runtime.Extensions.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Threading\src\System.Threading.csproj" />
+    <!-- error : CannotChangeAttribute : Attribute 'System.Diagnostics.SwitchLevelAttribute' on 'System.Diagnostics.BooleanSwitch' changed
+         from '[SwitchLevelAttribute(typeof(bool))]' in the implementation to '[SwitchLevelAttribute(typeof(Boolean))]' in the reference. -->
     <Reference Include="System.IO.FileSystem" />
```

Comment:
@tannergooding do you think the above mentioned ApiCompat issue will be fixed with your recent change to use primitive types always?

Reply:
This one I'm not sure is impacted. The logic I changed was only impacting `Boolean` itself using `bool` vs `Boolean` (and same for other primitives).

I wouldn't have expected it to also impact attribute decisions like this one. So there may still be another GenApi fix needed to ensure `typeof` here is allowing the keyword where possible.